### PR TITLE
test/validations

### PR DESCRIPTION
**Pull request by pjonatansr**
- added tests to validations

### CODE REVIEW

File: apps/web/src/middlewares/validations.test.ts [hunk 1]
```diff
@@ -0,0 +1,278 @@
+import { config } from 'config';
+import { NextApiRequest, NextApiResponse } from 'next';
+import withValidations from './validations';
+
+type ApiResponse = Partial<NextApiResponse> & {
+  message: string;
+};
+
+describe('Validations', () => {
```
Comment (sibelius) [inline apps/web/src/middlewares/validations.test.ts:9]: We can remove descobre 

Prefer to create a new test file instead of nesting

**Review by sibelius — APPROVED** [review]
Can we mock less?

**Review by sibelius — APPROVED** [review]
I prefer something like supertest

**Comment by pjonatansr** [comment]
> I prefer something like supertest

@sibelius,

should I rename the files to .spec.ts instead .test.ts? 
eg. validations.spec.ts

**Comment by sibelius** [comment]
The naming does not matter yet

**Pull request merged**